guides/theming.md

```diff
@@ -367,6 +367,83 @@ $my-palette: mat.define-palette(mat.$indigo-palette);
 }
 ```
 
+## Strong focus indicators
+
+By default, most components indicate browser focus by changing their background color as prescribed
+by the Material Design specification. This behavior, however, can fall short of accessibility
+requirements, such as [WCAG][], which require a stronger indication of browser focus.
```

Comment:
Intentionally omitted this link for now?

Reply:
This is actually a little markdown shortcut- if you leave the identifier here empty, it uses the text of the link (so this is equivalent to `[WCAG][WCAG]`

Reply:
The more ya know!